client.py

```diff
@@ -0,0 +1,197 @@
+
+    :param frame: The main frame.
+    :param output: An output of a command.
+    """
+
+    for widget in frame.winfo_children():
+        widget.destroy()
+    output = tkinter.Label(frame, text=f"output:\n{output}")
+    output.grid(row=0, column=0)
+    root.update()
+
+
+def run_download_command(command: List[str], frame: tkinter.Frame) -> None:
+    """
+    Handle download commands by passing the file's content to a shest server
+
+    :param command: A command given by the user.
+    :param frame: A frame for the client window configurations.
+    """
+
+    file_name = search("[a-zA-Z1-9]*\.[a-z]*", command[1]).group()
+    with requests.put(DOWNLOAD_PATH, data=encrypt_message(file_name),
+                      stream=True) as response:
+        with Path(command[1], 'w').open() as file:
+            for chunk in response.iter_content(chunk_size=8192):
+                file.write(decrypt_message(chunk).decode())
+
+    download_conf(frame, file_name)
+
+
+def run_upload_command(command: List[str], frame: tkinter.Frame) -> None:
+    """
+    Handle upload commands by passing the file's content to a shest server
+
+    :param command: A command given by the user.
+    :param frame: A frame for the client window configurations.
+    """
+
+    counter = 1
+    file_name = search("[a-zA-Z1-9]*\.[a-z]*", command[1]).group()
+    with Path(command[1], 'r').open() as file:
+        for chunk in file:
+            if counter == 1:
+                requests.put(UPLOAD_PATH + f'{file_name}?q=w',
+                             data=encrypt_message(chunk))
+            else:
+                requests.put(UPLOAD_PATH + f'{file_name}',
+                             data=encrypt_message(chunk))
+            counter += 1
+
+    upload_conf(frame, file_name)
+
+
+def run_regular_commands(command: List[str], frame: tkinter.Frame) -> None:
+    """
+    Handle regular commands by getting the output of the given them
+    from a remote shest server
+
+    :param command: A command given by the user.
+    :param frame: A frame for the client window configurations.
+    """
+
+    response = requests.put(REGULAR_PATH,
+                            data=encrypt_message(" ".join(command)))
+    output = decrypt_message(response.content).decode()
+
+    regular_conf(frame, output)
+
+
+def retrieve_input(inp_textbox: tkinter.Text, frame: tkinter.Frame) -> None:
+    """
+    Get a command input from the user and get the output to the command
+    from a fastapi server
+
+    :param inp_textbox: A textbox for the command input.
+    :param frame: A frame for the client window configurations.
+    """
+
+    command = inp_textbox.get("1.0", "end-1c").split(" ")
+    mapper = {DOWNLOAD_COMMAND: run_download_command,
+              UPLOAD_COMMAND: run_upload_command}
+    if command[0] in mapper.keys():
+        mapper[command[0]](command, frame)
+    else:
+        run_regular_commands(command, frame)
+    time.sleep(5)
+    root.destroy()
```

Comment:
Whenever I run a command, the output window is popped and it stays like that. I don't get returned to the input window again.